### ISSUE TITLE
Preserve the original order of copyrights in SPDX output

### ISSUE THREAD

**Issue by armintaenzertng** [nexB/scancode-toolkit#3457]
### Description
Based on [this comment](https://github.com/nexB/scancode-toolkit/pull/3456#discussion_r1265476064):
> [In SPDX output,] we should not use a set [for copyrights] and instead preserve the original order of copyrights: it matters.

**Comment by pombredanne** [comment]
FWIW, the order of copyrights does matter as the first one is usually the primary copyright. To keep the insertion order, you can use a list and at last do something like:
```
def unique(objects):
    """
    Return a list of unique objects.
    """
    uniques = []
    for obj in objects:
        if obj not in uniques:
            uniques.append(obj)
    return uniques
```
Or do the same on the fly. Or use an ordered set. Or a use a dict using only keys.

**Comment by mjherzog** [comment]
I agree that we should keep the original order, but there are no standards here. In general the first copyright notice is/should be the most recent but that is different from primary. In many cases the most recent copyright notice is for a minor change.
